package.json

```diff
@@ -159,13 +160,12 @@
     "eslint-plugin-react": "^5.2.2",
     "fakeredis": "^2.0.0",
     "foreman": "^1.4.1",
-    "jest": "^22.1.4",
```

Comment:
don't we still need jest for the e2e tests?  Lines 11-15 still call jest.  Happy for them to move too -- but also make sure the docs don't mention jest

Reply:
our test scripts use `jest`

Reply:
ok, sounds good -- can we make a comment for that, at least.  I'm ++ for a downstream dependency, but just to avoid the perception of magic, maybe add a comment above the jest script lines?

Is it still impossible to add comments to package.json?

Reply:
Still impossible... but for posterity here is some more information.

This is the documentation for create-react-app testing, which is what we're using:

https://facebook.github.io/create-react-app/docs/running-tests

The front page of [create-react-app](https://facebook.github.io/create-react-app/) says:

> Your app only needs one build dependency. We test Create React App to make sure that all of its underlying pieces work together seamlessly – no complicated version mismatches.

So to get the benefits of cra, we want to stay out of the business of manually managing dependencies that it already declares, and only [bump the react-scripts version](https://facebook.github.io/create-react-app/docs/updating-to-new-releases) when it's time to upgrade.